Project-Aurora/Project-Aurora/Devices/DeviceManager.cs

```diff
@@ -96,23 +96,24 @@ public int RetryAttempts
-            devices.Add(new DeviceContainer(new Devices.UnifiedHID.UnifiedHIDDevice()));     // UnifiedHID Device
-            devices.Add(new DeviceContainer(new Devices.Wooting.WootingDevice()));           // Wooting Device
-            devices.Add(new DeviceContainer(new Devices.Creative.SoundBlasterXDevice()));    // SoundBlasterX Device
-            devices.Add(new DeviceContainer(new Devices.LightFX.LightFxDevice()));           //Alienware
-            devices.Add(new DeviceContainer(new Devices.Dualshock.DualshockDevice()));       //DualShock 4 Device
-            devices.Add(new DeviceContainer(new Devices.Drevo.DrevoDevice()));               // Drevo Device
-            devices.Add(new DeviceContainer(new Devices.YeeLight.YeeLightDevice()));         // YeeLight Device
-            devices.Add(new DeviceContainer(new Devices.Asus.AsusDevice()));               // Asus Device
-            devices.Add(new DeviceContainer(new Devices.NZXT.NZXTDevice()));                 //NZXT Device
+            devices.Add(new DeviceContainer(new Devices.Logitech.LogitechDevice()));         // Logitech
+            devices.Add(new DeviceContainer(new Devices.Corsair.CorsairDevice()));           // Corsair
+            devices.Add(new DeviceContainer(new Devices.Razer.RazerDevice()));               // Razer
+            devices.Add(new DeviceContainer(new Devices.Roccat.RoccatDevice()));             // Roccat
+            devices.Add(new DeviceContainer(new Devices.Clevo.ClevoDevice()));               // Clevo
+            devices.Add(new DeviceContainer(new Devices.CoolerMaster.CoolerMasterDevice())); // CoolerMaster
+            devices.Add(new DeviceContainer(new Devices.AtmoOrbDevice.AtmoOrbDevice()));     // AtmoOrb Ambilight
+            devices.Add(new DeviceContainer(new Devices.SteelSeries.SteelSeriesDevice()));   // SteelSeries
+            devices.Add(new DeviceContainer(new Devices.UnifiedHID.UnifiedHIDDevice()));     // UnifiedHID
+            devices.Add(new DeviceContainer(new Devices.Wooting.WootingDevice()));           // Wooting
+            devices.Add(new DeviceContainer(new Devices.Creative.SoundBlasterXDevice()));    // SoundBlasterX
+            devices.Add(new DeviceContainer(new Devices.LightFX.LightFxDevice()));           // Alienware
+            devices.Add(new DeviceContainer(new Devices.Dualshock.DualshockDevice()));       // DualShock 4
+            devices.Add(new DeviceContainer(new Devices.Drevo.DrevoDevice()));               // Drevo
+            devices.Add(new DeviceContainer(new Devices.YeeLight.YeeLightDevice()));         // YeeLight
+            devices.Add(new DeviceContainer(new Devices.Asus.AsusDevice()));                 // Asus
+            devices.Add(new DeviceContainer(new Devices.NZXT.NZXTDevice()));                 // NZXT
+            devices.Add(new DeviceContainer(new Devices.Hue.HueDevice()));                   // Philips Hue
```

Comment:
There's really no need for these sorts of very minor cosmetic changes, especially when it falls outside of the scope of your PR.

Reply:
Honestly those comments don't need to be there _as mentioned in the discord_ since we can clearly see what is what